app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,25 @@
+class SessionsController < ApplicationController
+  before_action :logged_in_user, only: [:new]
+
+  def new
```

Comment:
Take a look at the style guide for recommendations on writing methods with no body